### ISSUE TITLE
hasChanges depends on user language

### ISSUE THREAD

**Issue by jnugh** [czproject/git-php#25]
I noticed that `hasChanges` compares the shell output in order to determine if there are changes
```php
return (strpos($lastLine, 'nothing to commit')) === FALSE;
```

Some users have other locales installed:
```
git status                                                                                                                                                                    [master] 
Auf Branch master
Ihr Branch ist auf dem selben Stand wie 'origin/master'.

nichts zu committen, Arbeitsverzeichnis unverändert
```
`nichts zu committen` is the German translation of `nothing to commit` which is not beeing detected correctly.

I had a look at the code and did not find a similar string comparison, changing 
```php
$lastLine = exec('git status');
// to
$lastLine = exec('LANG=en git status');
```

works for me, I'm not sure if this works on windows. If you'd like I could open a PR for this - if there are other places where shell output is compared I would like to fix those as well if you could point me there. Updating `processCommand` would probably work in most cases if there are any.

My workaround for now is to run my application with `LANG=en`.

**Comment by janpecha** [comment]
Hi, thanks for report. I changed `git status` to `git status --porcelain` - it should help.

Please try current `master` version, if it be ok, I will tag new release.

**Comment by jnugh** [comment]
HI @janpecha pretty close but I think this change breaks it for everyone else as well:
`hasChange` needs to return `true` if `git status` returned something.
It works fine like this:
```php
return !empty($output);
```

**Comment by janpecha** [comment]
Of course, you are right. It was for tests, I forget change it back to `return !empty()`. Try it now, please.

**Comment by jnugh** [comment]
Works, thank you very much for the quick fix. :+1:

**Comment by janpecha** [comment]
Great, I just released new version 3.14.0. Thanks for help!
